Jenkinsfile

```diff
@@ -0,0 +1,91 @@
+pipeline {
+    agent {
+        label 'linux-large'
+        //TODO 6146 windows builds ?using label 'proxmox-windows'?
```

Comment:
might be able to use 'node' syntax so you can have the windows and linux work in parallel.